.eslintrc.json

```diff
@@ -17,6 +17,9 @@
         "mocha": true
     },
     "globals": {
+        "process": false,
```

Comment:
I dont want we add tis globally, there is some syntax to enable or disable some rules with eslint. It's better to use that instead of enabling globals names on the whole project.
check here http://eslint.org/docs/rules/no-undef#node